### PR TITLE
Thilini

### PR DESCRIPTION
**Pull request by ThiliniUdari**

### CODE REVIEW

File: src/app/Main/customer-about-us/customer-about-us.component.html [hunk 1]
```diff
@@ -1 +1,27 @@
-<p>customer-about-us works!</p>
+<div class="" >
```
Comment (BLasan) [inline src/app/Main/customer-about-us/customer-about-us.component.html:1]: Try to develop the code inside <div class="main-content> <div class="container-fluid> </div> </div>
@ThiliniUdari  Upload a screenshot of the components you have designed

Comment (ThiliniUdari) [reply src/app/Main/customer-about-us/customer-about-us.component.html:1]: Screenshot_2020-05-17 Material Dashboard Angular by Creative Tim(1)
Screenshot_2020-05-17 Material Dashboard Angular by Creative Tim
Screenshot_2020-05-17 Material Dashboard Angular by Creative Tim(1)
Screenshot_2020-05-17 Material Dashboard Angular by Creative Tim(2)

**Comment by BLasan** [comment]
@ThiliniUdari  Refer these link: https://demos.creative-tim.com/material-dashboard-angular2/#/documentation/tutorial
https://material.angular.io/

**Comment by ThiliniUdari** [comment]
Added the missed classes and removed files.

**Pull request merged**